docs/AMF/using-amf/resolution.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: amf_resolution
+title: Resolution
+---
+
+Resolution is the process of applying multiple transformations to the AMF Graph model. The complete resolution pipeline has the following steps:
+
+![resolution pipeline](../../assets/images/amf/resolution-pipeline.png)
+
+The most common use case of resolution is to flatten and simplify the model by:
+- Resolving links and inheritances
+- Applying traits and resourceTypes (in the case of RAML)
+- Normalization of shapes and parameters
+
+Another use case where resolution is of utmost importance is when making a conversion from one API specification to another.
+For example, when converting a RAML API to OAS, multiple verifications and conversions are made in order to have a valid, OAS-compliant API.
```

Comment:
not quite right, the resolution stage actively changes the WebApi model to insert certain fields or change others so that the resulting API is valid in the target spec